sys/mips/include/atomic.h

```diff
@@ -90,20 +90,14 @@ static __inline int atomic_fcmpset_8(__volatile uint8_t *, uint8_t *, uint8_t);
 static __inline int atomic_cmpset_16(__volatile uint16_t *, uint16_t, uint16_t);
 static __inline int atomic_fcmpset_16(__volatile uint16_t *, uint16_t *, uint16_t);
 
-static __inline void
```

Comment:
I'd split the QEMU_TLB_WORKAROUND32 removal out into another commit

Reply:
done